tests/lifecycle/tests/lifecycle_termination_grace_period.go

```diff
@@ -0,0 +1,107 @@
+package tests
+
+import (
+	. "github.com/onsi/ginkgo"
+	. "github.com/onsi/gomega"
+	"github.com/test-network-function/cnfcert-tests-verification/tests/globalhelper"
+	"github.com/test-network-function/cnfcert-tests-verification/tests/globalparameters"
+	"github.com/test-network-function/cnfcert-tests-verification/tests/lifecycle/lifehelper"
+	"github.com/test-network-function/cnfcert-tests-verification/tests/lifecycle/lifeparameters"
+	"github.com/test-network-function/cnfcert-tests-verification/tests/utils/deployment"
+	"github.com/test-network-function/cnfcert-tests-verification/tests/utils/namespaces"
+	"k8s.io/utils/pointer"
+)
+
+var _ = Describe("lifecycle lifecycle-termination-grace-period", func() {
+
+	BeforeEach(func() {
+		By("Clean namespace before each test")
+		err := namespaces.Clean(lifeparameters.LifecycleNamespace, globalhelper.APIClient)
+		Expect(err).ToNot(HaveOccurred())
+	})
+
+	// 47399
+	It("One deployment, one pod, terminationGracePeriodSeconds is not set", func() {
+
+		By("Define deployment without terminationGracePeriodSeconds")
+		deploymentStruct, err := lifehelper.DefineDeployment(false, 1, 1, "lifecycleput")
+		Expect(err).ToNot(HaveOccurred())
+
+		deploymentStruct = lifehelper.RemoveterminationGracePeriod(deploymentStruct)
+
+		err = globalhelper.CreateAndWaitUntilDeploymentIsReady(deploymentStruct, lifeparameters.WaitingTime)
+		Expect(err).ToNot(HaveOccurred())
+
+		By("Start lifecycle lifecycle-pod-termination-grace-period test")
+		err = globalhelper.LaunchTests(
+			[]string{lifeparameters.LifecycleTestSuiteName},
+			lifeparameters.SkipAllButTerminationGracePeriodRegex)
+		Expect(err).ToNot(HaveOccurred())
+
+		By("Verify test case status in Junit and Claim reports")
+		err = globalhelper.ValidateIfReportsAreValid(
+			lifeparameters.TerminationGracePeriodDefaultName,
+			globalparameters.TestCasePassed)
```

Comment:
This TC passes on 3.2 but will fail on 3.3 when it's released because of this PR https://github.com/test-network-function/test-network-function/pull/562
With that PR, we can detect whether the terminationGracePeriod value was actually set by the user (no matter which value), which was the original aim of this TNF test case.